test_regress/t/t_debug_emitv.py

```diff
@@ -9,23 +9,30 @@
 
 import vltest_bootstrap
 
-test.scenarios('vlt')
+test.scenarios("vlt")
 
 test.lint(
     # We also have dump-tree turned on, so hit a lot of AstNode*::dump() functions
     # Likewise XML
     v_flags=["--lint-only --dumpi-tree 9 --dumpi-V3EmitV 9 --debug-emitv"])
 
-output_v = test.glob_one(test.obj_dir + "/" + test.vm_prefix + "_*_width.tree.v")
+output_vs = test.glob_some(test.obj_dir + "/" + test.vm_prefix + "_*_width.tree.v")
```

Comment:
I don't know why I get two `width.tree.v` files from this test now, but I do.  In any case, they are the same so I'm currently just asserting that they all match the golden file.